avocado/plugins/xunit.py

```diff
@@ -35,7 +36,8 @@ def configure(self, parser):
 
         self.parser = parser
         run_subcommand_parser.output.add_argument(
-            '--xunit', type=str, dest='xunit_output', metavar='FILE',
```

Comment:
Yep.  Drecreasing the selftest execution time is something I don't mind at all.  But maybe leaving at least one is a good thing, since the test occurs at two very different levels.  I'll take a better look at those functional tests and send a proposal on v2.